awx/main/models/unified_jobs.py

```diff
@@ -817,14 +794,17 @@ def parent_instance_set(key, val):
             if self.status in ('pending', 'waiting', 'running'):
```

Comment:
@kdelee has a proposal forming which would boot `'waiting'` from this list. This is slightly a behavior change, but the ask is that current_job is not set by the task manager itself.

I'm very supportive of that proposal.